bcl.sln

```diff
@@ -1,6 +1,6 @@
 
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
```

Comment:
Just checking - did you make sure this works in VS? The .sln parser is notorious for being very bad and subtly broken, things like dependencies get dropped based on whitespace

Reply:
You mean VSMac or VSWin? for the latter there should be no change as the .sln will be checked out with CRLF on Windows.

Reply:
OK, wasn't sure whether our sln parser was bug-for-bug compatible